Project.toml

```diff
@@ -1,11 +1,12 @@
 name = "AdvancedPS"
 uuid = "576499cb-2369-40b2-a588-c64705576edc"
 authors = ["TuringLang"]
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Officially, this is not breaking since currently nothing is exported. However, I know that this will break Turing since we call some of the internal methods without RNG, so I thought it would be safer to declare it as breaking.